package.json

```diff
@@ -14,6 +14,7 @@
   "dependencies": {
     "@bcrs-shared-components/mixins": "^1.1.16",
     "@mdi/font": "^5.9.55",
+    "@vuelidate/core": "^2.0.3",
```

Comment:
Since you're changing this file, please update the patch version on line 3.

Reply:
Wait - this file is used by Storybook. Does SB need this package, or is it needed by the component only?

Reply:
Yes, scope this import to the component itself.